package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "axios": "^0.11.0",
+    "babel-register": "^6.8.0",
```

Comment:
My machine was complaining about this, does this need to be in dependencies or devDep?